src/models/pie.js

```diff
@@ -205,7 +205,9 @@ nv.models.pie = function() {
                 }
 
                 if (donutLabelsOutside) {
-                    labelsArc = d3.svg.arc().outerRadius(arc.outerRadius()).startAngle(startAngle).endAngle(endAngle);
+                    labelsArc = d3.svg.arc().outerRadius(arc.outerRadius());
+                    if (startAngle !== false) labelsArc.startAngle(startAngle);
+                    if (endAngle !== false) labelsArc.endAngle(endAngle);
```

Comment:
Well, the logic should match, so we can either set these to be if(startAngle) or change the other locations to check for !== false, up to you.